en/orm/associations.rst

```diff
@@ -357,6 +357,23 @@ syntax::
         }
     }
 
+Sometimes you may want to configure composite keys in your associations::
+
+    $this->hasMany('Reviews', [
+        'foreignKey' => [
+            'article_id',
+            'article_name'
+        ],
+        'bindingKey' => [
+            'article_id',
+            'article_name'
+        ]
+    ]);
+
```

Comment:
Well in the example above the bindingKey would "most likely" already be setup like that. 
However I had a short hard time understanding your example (but it could be me!).